markdown/en/api/ff-navigation.api.md

```diff
@@ -3,35 +3,35 @@ ___
 ### Properties
 | Name | Description |
 | ---- | ----------- |
-|**layout**&nbsp;(String) **Options**:&nbsp;vertical,&nbsp;horizontal (default: horizontal)| Defines the layout of the Navigation.|
-|**flyout**&nbsp;(Boolean) **Options**:&nbsp;true,&nbsp;false (default: true)| Defines if the sub-elements should flyout or should extend inside the parent.|
-|**mobile**&nbsp;(Boolean) **Options**:&nbsp;true,&nbsp;false (default: false)| When true, the navigation element will be visible.|
-|**fetch-initial**&nbsp;(String) **Options**:&nbsp;true,&nbsp;false (default: true)|If true (default), the navigation data will be requested on the first attachment to the DOM.|
-|**first-fetch**&nbsp;(Number)| Defines up to which start level the navigation data for the initial loading should be requested. **Note**: Navigation data may be up to 1MB and can take his time to load. The first impression for the user when he needs to wait for the initial page load may be not so pleasant.
-|**fetch-size**&nbsp;(Number)| Defines how many level of the following navigation data should be loaded in subsequent requests. Fewer level -> more requests and inversely more level -> more data per request.|
-|**max-fetch**&nbsp;(Number)| Defines up to which level of the navigation hirachy the data should be loaded.|
-|**fetch-time**&nbsp;(Number)| Defines the time between navigation requests. (Default: 100ms)|
+| **layout**&nbsp;(String) **Options**:&nbsp;"vertical",&nbsp;"horizontal" (default: "horizontal") | Defines the layout of the Navigation. |
+| **flyout**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "true") | Defines if the sub-elements should fly out or should expand inside the parent. |
+| **mobile**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "false") | If true, the navigation element will be visible. |
+| **fetch-initial**&nbsp;(String) **Options**:&nbsp;"true",&nbsp;"false" (default: "true") | If true, the navigation data will be requested on the first attachment to the DOM. |
+| **first-fetch**&nbsp;(Number) (default: 2) | Defines up to which start level the navigation data for the initial loading should be requested. **Note**: Navigation data may be up to 1MB and can take his time to load. The first impression for the user when he needs to wait for the initial page load may be not so pleasant. |
```

Comment:
Navigation data may be up to 1MB and can take ~~his~~ _its_ time to load

Reply:
Maybe the data identifies as male?